### PR TITLE
Restore VLT Training

### PR DESCRIPTION
**Pull request by stevepiercy**
Replaces #867, which had a strange merge.

----
📚 Documentation preview 📚: https://plone-training--880.org.readthedocs.build/

### CODE REVIEW

**Comment by stevepiercy** [comment]
@danalvrz all right, I don't know what happened, so I refactored the directory name to match the training on https://2024.ploneconf.org/en/schedule/training/training-customizing-the-volto-light-theme, then pushed. Now the files are back and we have an open PR with a pull request preview build. I still have not reviewed it, as the naming confusing.

File: docs/customizing-volto-light-theme/installing-vlt.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
++const theme = '@kitconcept/volto-light-theme';
+
+ module.exports = {
+   addons,
+-  theme
++  theme,
+ };
+ ```
+
+That's it! Your project should now be using Volto Light Theme with its additional blocks and components.
```
Comment (stevepiercy) [inline docs/customizing-volto-light-theme/installing-vlt.md:71]: Do you need to restart the instance, or is that handled automatically?

**Comment by stevepiercy** [comment]
@danalvrz I'm done with my review. There are critical issues with installation that must be fixed before this is merged. I strongly suggest not reinventing the wheel and just link to our docs, unless you have a Very Good Reason™ not to do so.

File: docs/customizing-volto-light-theme/installing-vlt.md (hunk 1 above)
Comment (danalvrz) [reply docs/customizing-volto-light-theme/installing-vlt.md:73]: yes you do, I just added the comment, thanks.

File: docs/customizing-volto-light-theme/integrate-new-block.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "@kitconcept/volto-highlight-block",
+  "@kitconcept/volto-introduction-block",
+  "@kitconcept/volto-separator-block",
+  "@kitconcept/volto-slider-block",
+  "@plone-collective/volto-relateditems-block",
+  "@kitconcept/volto-light-theme"
+],
+```
+
+That's it! Your project should now be using `@plone-collective/volto-relateditems-block`, which shows related items for the content as a list of links.
```
Comment (stevepiercy) [inline docs/customizing-volto-light-theme/integrate-new-block.md:38]: Does this step also require a restart to see the changes?

**Review by stevepiercy — COMMENTED** [review]
This looks good, with just one comment about a potential missing step to restart to see changes in another place.

**Comment by stevepiercy** [comment]
@danalvrz I just sent you an invitation to give you access to merge this PR, once you've addressed my final comment. Please accept the invitation. Overall LGTM.

File: docs/customizing-volto-light-theme/integrate-new-block.md (hunk 2 above)
Comment (danalvrz) [reply docs/customizing-volto-light-theme/integrate-new-block.md:44]: it does, I just added the comment. Thanks!

**Pull request merged**